### PR TITLE
[TDX-2195] Dev portal app reg with multi-runtime group support

### PR DESCRIPTION
**Pull request by Mierenga**
### Summary

This describes an update to dev portal's app registration. It was previously available only in the `default` runtime group. App registration is now available as a `Tech Preview` in all runtime groups. However, for compatibility with Kong Gateway's < 3.0, there are some differences with how it is applied, depending on the runtime group.

### Reason

https://konghq.atlassian.net/browse/TDX-2195
Application registration is opening support for any runtime group. Only compatible with Kong Gateway versions >= 3.0

### CODE REVIEW

**Comment by cloudjumpercat** [comment]
@Mierenga Will you be adding additional changes or is this ready for a copy edit? Thanks!

**Comment by Mierenga** [comment]
@cloudjumpercat There will be additional changes in the next couple weeks before it is ready for docs review, thanks for checking in!

**Comment by cloudjumpercat** [comment]
@Mierenga Hi! Just checking in to see if you have any more changes for this PR. Thanks!

**Comment by Mierenga** [comment]
@cloudjumpercat This info is ready for review from the docs team, thanks!

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 1]
```diff
@@ -99,3 +119,11 @@ To remove a plugin by disabling application registration, follow these steps:
 You can
 [re-enable application registration](/konnect/dev-portal/applications/enable-app-reg)
 at any time.
+
+### Differences between runtime groups
+
+If you need to use version of {{site.base_gateway}} less than 3.0, you must create your service version in the `default` runtime group. Non-default runtime groups are only compatible with {{site.base_gateway}} 3.0 and higher.
```
[the remark this answers is not in the document]
Comment (Mierenga) [reply app/konnect/dev-portal/applications/enable-app-reg.md]: updated in https://github.com/Kong/docs.konghq.com/pull/4473/commits/78fdf9479b4b741130cc9e3ae0f4d52339967cea

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 2]
```diff
@@ -63,20 +75,28 @@ service, and follow these steps:
 
 4. Click **Enable**.
 
-    All versions of this service now include
-    read-only entries for the  `acl` and `oidc` plugins.
+    This versions of this service packages now includes
+    read-only entries for the  `konnect-application-auth` and `openid-connect` plugins.
```
[the remark this answers is not in the document]
Comment (Mierenga) [reply app/konnect/dev-portal/applications/enable-app-reg.md]: updated in https://github.com/Kong/docs.konghq.com/pull/4473/commits/78fdf9479b4b741130cc9e3ae0f4d52339967cea

**Review by nateslo — APPROVED** [review]
This looks good to me from a technical standpoint

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 3]
```diff
@@ -4,16 +4,30 @@ content_type: how-to
 ---
 
 To grant developers access to [register an application](/konnect/dev-portal/applications/dev-reg-app-service), you must enable application registration for a service version.
-When you enable application registration, {{site.konnect_saas}} enables two plugins automatically: [ACL](/hub/kong-inc/acl), and your choice of [Key Authentication](/hub/kong-inc/key-auth)
-or [OIDC](/hub/kong-inc/openid-connect). These plugins run in the background to support application registration for the service and are managed by
+When you enable application registration, {{site.konnect_saas}} enables plugins automatically to support the desired mode, either key authentication or OpenID Connect.
```
Comment (codelemur) [inline app/konnect/dev-portal/applications/enable-app-reg.md:7]: Nit:
```suggestion
When you enable application registration, {{site.konnect_saas}} automatically enables plugins to support the desired mode, either [Key Authentication](https://docs.konghq.com/hub/kong-inc/key-auth/) with [ACL](https://docs.konghq.com/hub/kong-inc/acl/) or [OpenID Connect](https://docs.konghq.com/hub/kong-inc/openid-connect/).
```

I just want to be more explicit in the explanations, if that makes sense.

Comment (codelemur) [reply app/konnect/dev-portal/applications/enable-app-reg.md:7]: Please disregard!

File: app/konnect/dev-portal/applications/enable-app-reg.md [hunk 4]
```diff
@@ -4,16 +4,30 @@ content_type: how-to
...
+{:.note}
+> **Note:** The `default` runtime group is the one that is first created in each region when you create an organization. Although it can be renamed, it will always be the oldest runtime group in the region. See [default runtime group](/konnect/runtime-manager/runtime-groups/#default-runtime-group) for additional context.
+
 ## Prerequisites
 
 - A service that is versioned and published to the
   {{site.konnect_short_name}} Dev Portal so that it appears in the catalog.
 
-- The service version must be in the `default` runtime group.
+- The service version can be in any runtime group.
```
Comment (codelemur) [inline app/konnect/dev-portal/applications/enable-app-reg.md:25]:
```suggestion
- The service version can be in any runtime group, as long as the following conditions are met:
```

**Review by codelemur — APPROVED** [review]
LGTM

**Review by Guaris — APPROVED** [review]

**Comment by Guaris** [comment]
I'm pushing this to a feature branch with this PR
https://github.com/Kong/docs.konghq.com/pull/4716

So they can both go out together on Monday

**Pull request merged**